catalog/geoscience-aus-cat/projects_geoscience-aus-cat_assets_ga_ls_landcover_class_cyear_2.jsonnet

```diff
@@ -32,7 +32,13 @@ local self_url = catalog_subdir_url + base_filename;
   title: 'DEA Land Cover ' + version,
   'gee:type': ee_const.gee_type.image_collection,
   description: |||
-    DEA Land Cover provides annual land cover classifications for Australia using the FAO (Food and Agriculture Organisation)'s Land Cover Classification System taxonomy Version 2 (Di Gregorio and Jansen, 1998; 2005).
+    Digital Earth Australia (DEA) Land Cover translates over 30 years of satellite imagery into evidence of how Australia's land, vegetation and waterbodies have changed over time.
+    
+    Land cover is the observed physical cover on the Earth's surface including trees, shrubs, grasses, soils, exposed rocks, water bodies, plantations, crops and built structures. A consistent, Australia-wide land cover product helps understanding of how the different parts of the environment change and inter-relate. Earth observation data recorded over a period of time firstly allows the observation of the state of land cover at a specific time and secondly the way that land cover changes by comparison between times.
+    
+    DEA Land Cover provides annual land cover classifications for Australia using the Food and Agriculture Organisation Land Cover Classification System taxonomy Version 2 (Di Gregorio and Jansen, 1998; 2005).
```

Comment:
We recommend that the first paragraph contain the most specific information to tell people what the dataset is. Could you move the preceding two paragraphs to the bottom of the description?